src/common/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Link } from 'react-router-dom'
+import { BrowserRouter as Router, Route } from 'react-router-dom'
```

Comment:
You are only using Route from react-router-dom, you can remove BrowserRouter as Router. Or shortcut: **option+shif+o** this will sort and clean your imports automatically.